### DISCUSSION TITLE
Is it possible to just see the BMS data in GX, and disable driver settings?

### DISCUSSION THREAD

**Discussion by LecramRed** [Louisvdw/dbus-serialbattery#517] [Q&A] [answered]
I just want to have the JK BMSses data visible in GX, and I want to have the GX control over the (dis)charging processes.

When I install the driver, the driver disables the settings in GX and uses its own settings to control (dis)carging. How can I disable this and just have the data visible in GX?

**Comment by mr-manuel** [comment, marked as answer]
There are already a few discussions about this.

- https://github.com/Louisvdw/dbus-serialbattery/discussions/482
- https://github.com/Louisvdw/dbus-serialbattery/discussions/492
- https://github.com/Louisvdw/dbus-serialbattery/discussions/490
